src/js/modules/comments.js

```diff
@@ -0,0 +1,44 @@
+import commentsCounter from './comment_counter';
+
+const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const appID = '9JcQwe8YeiV9ciPMfpK9';
```

Comment:
The app_id is a type of secret information that should not be committed into the public repository, it's highly recommended to put such a thing in a `.env` file and add enough description on the Readme file about where and how to create and use one.